spec/test_measures/ChangeBuildingLocation/measure.rb

```diff
@@ -119,7 +119,7 @@ def run(model, runner, user_arguments)
         return false
       end
       if stat_files.size == 0
-        runner.registerError("Cound not find the stat file in the EPW directory")
+        runner.registerError("Count not find the stat file in the EPW directory")
```

Comment:
@kflemin This seems like intentional change? Just a comment in test measure so doesn't matter too much

Reply:
@DavidGoldwasser this is a typo fix